### DISCUSSION TITLE
Possible to install EOS8 on usb drive ?

### DISCUSSION THREAD

**Discussion by quadibus** [elementary/triage#671] [Q&A]
Hello,
Before testing it, I would like to know if it is possible to install EOS8 on usb drive. I'm not talking about install FROM usb ( live usb key), but install EOS8 on usb drive instead of install on internal drive ? 
I already do it for ubuntu 24 but I would switch to EOS8.

Thanks a lot
Quad.

**Comment by jeremypw** [comment]
It will install and run faster if you install onto a VM.  I use Virtual Machine Manager but others should work.  You should choose Ubuntu 24.04 as the VM type but source it from the ISO you downloaded. There are a few differences in handling gestures for example, and the network *appears* as only Ethernet (although the host is wireless) but in general its pretty close to a native experience.  Installing on a USB will run much more slowly.  I've not tried it with OS8.
